index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio Website</title>
+</head>
+<body>
+    <div id="wrapper">
+        <header id="container">
+            <div>
+                <p><a class="logo" href="#">Suleiman</a></p>
+            </div>
+            <div id="navbar">
+                <a href="#"><img src="images/Hamburguer.png" alt="an image here" class="image"></a>
+            </div>
+        </header>
+        
+        <section>
+            <h1 id="name">I'm Suleiman Welcome To My Portfolio Website</h1>
+        
+             <p id="about">I'm a software developer and i can help you build a solution, implement a feature or build 
+                a website for your business, organization or firm as the case may be. feel free to connect
+                with me using the social media links.
+             </p>
+        
+        
+            <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
+        
+            <ul class="socialmedia">
+                <li><a href="#"><img src="images/twiter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/Linkedin.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/m.svg" alt="m"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/manito.svg" alt="magnito"></a></li>
+            </ul>
+        </section>
```

Comment:
- [ ] Kindly fix the margin of this section as requested by the previous reviewer. You can start removing the 8px margin around the body by setting `margin: 0;` on the `body` element. It will help you in removing the horizontal overflow.
Currently, it is looking like this at 375px
<img src="https://user-images.githubusercontent.com/17243325/205217982-dce8b064-ef6d-4f03-a5ed-2cf138ffe1fe.png" width="300">